nixos/modules/services/security/reaction.nix

```
@@ -177,11 +171,29 @@ in
               lib.optionalString (cfg.loglevel != null) " -l ${cfg.loglevel}"
             }
           '';
-          StateDirectory = "reaction";
-          RuntimeDirectory = "reaction";
-          WorkingDirectory = "/var/lib/reaction";
 
-          CapabilityBoundingSet = [ "CAP_NET_ADMIN" ];
+          NoNewPrivileges = true;
+
+          RuntimeDirectory = "reaction";
+          RuntimeDirectoryMode = "0750";
+          WorkingDirectory = "%S/reaction";
```

Just to be sure, it's probably better to hard code since this can presumably break if `StateDirectory` is a list, according to @ju1m.


```suggestion
          WorkingDirectory = "/var/lib/reaction";
```